asserts/store_asserts.go

```diff
@@ -0,0 +1,94 @@
+package asserts
+
+import (
+	"fmt"
+	"net/url"
+)
+
+// EnterpriseStore holds an enterprise-store assertion, defining the
+// configuration needed to connect a device to the enterprise store.
+type EnterpriseStore struct {
+	assertionBase
+	address *url.URL
+}
+
+// OperatorID returns the account id of the enterprise store's operator.
+func (estore *EnterpriseStore) OperatorID() string {
+	return estore.HeaderString("operator-id")
+}
+
+// Store returns the identifying name of the operator's enterprise store.
+func (estore *EnterpriseStore) Store() string {
+	return estore.HeaderString("store")
```

Comment:
I see @pedronis is wondering about this term. Do you know why?  It sounds like this matches what was used in the model already?